docs/_partners/common-migration-errors.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Common Errors When Migrating Users
+order: 11
+layout: post-toc
+redirect_from: /partners/
+---
+<a id="deploy-errors"></a>
+
+## Common Errors When Migrating Users and Zaps Between Versions
+
+There are certain changes to an integration the platform considers breaking changes, meaning migrating users and their Zaps to the new version will break existing, live Zaps and cause them to be turned off. 
+
+Zapier's platform runs basic automated checks to prevent situations that may break Zaps before migrating users. But the checks can't cover everything, so please be extra aware of possible breaking changes when updating your integrations. No checks exist for version promotions, as having existing incompatible versions in parallel is acceptable.
+
+Here are some commonly encountered errors and breaking changes to watch out for, with suggested solutions:
+
+> **Note:** You can also always choose not to migrate users, and follow our [deprecation process](https://platform.zapier.com/cli_tutorials/versions#deprecate-an-older-version-of-your-integration).
```

Comment:
Strictly speaking, they can just push an incompatible version, promote it, and do nothing. If the old version will keep working (same API, etc) then that's also fine. It makes support a little tougher, but it's definitely an option.